docs/freebsd/README.md

```diff
@@ -89,7 +93,34 @@ Then, to permit remote access to the VM, you must configure DHCP and enable `ssh
 # sysrc ifconfig_DEFAULT=DHCP
 ```
 
-If you plan to run the syscall executor as root, ensure that root SSH logins are permitted by adding `PermitRootLogin without-password` to `/etc/ssh/sshd_config`.  Otherwise, create a new user with `adduser`.  Install an ssh key for the user and verify that you can SSH into the VM from the host.
+If you plan to run the syscall executor as root, ensure that root SSH logins are permitted by adding `PermitRootLogin without-password` to `/etc/ssh/sshd_config`.  Otherwise, create a new user with `adduser`.  Install an ssh key for the user and verify that you can SSH into the VM from the host.  Note that bhyve requires the use of the root user for the time being.
+
+### Running Under bhyve
+
+Some additional steps are required on the host in order to use bhyve.  First, ensure that the host system is at r346550 or later.  Second, since bhyve currently does not support disk image snapshots, ZFS must be used to provide equivalent functionality.  Create a ZFS data set and copy the VM image there.  The data set can also be used to store the syzkaller workdir.
+```console
+# zfs create data/syzkaller
+# cp FreeBSD.raw /data/syzkaller
```

Comment:
For consistency, you might want to use `FreeBSD-13.0-CURRENT-amd64.raw` instead of `FreeBSD.raw`.

Reply:
Done, thanks.